apps/website/src/components/ExcerptText.tsx

```diff
@@ -20,8 +21,22 @@ export interface ExcerptTextProps {
 export function ExcerptText({ model, excerpt }: ExcerptTextProps) {
 	return (
 		<>
-			{excerpt.spannedTokens.map((token) => {
+			{excerpt.spannedTokens.map((token, idx) => {
 				if (token.kind === ExcerptTokenKind.Reference) {
+					const source = token.canonicalReference?.source;
+
+					if (source && 'packageName' in source && source.packageName === 'discord-api-types') {
+						const base = 'https://discord-api-types.dev/api/discord-api-types-v10';
```

Comment:
This should probably be moved into a constants file.

Reply:
Yes